src/tools/PrizeTierController/EditPrizeTiersModal/index.tsx

```diff
@@ -0,0 +1,162 @@
+//             ...updatedPrizeTierEdits[prizePool.chainId][prizePool.prizeTierHistoryMetadata.address],
+//             ...newPrizeTierEdits
+//           }
+//         })
+//         return updatedPrizeTierEdits
+//       })
+//       setIsEditPrizeTierModalOpen(false)
+//     },
+//     [decimals, setPrizeTierEdits, setIsEditPrizeTierModalOpen, prizePools]
+//   )
+
+//   return (
+//     <div>
+//       <p>Make changes across all Prize Tier History configurations at once.</p>
+//       <EditPrizeTierHistoryForm onSubmit={onSubmit} />
+//     </div>
+//   )
+// }
+
+const SimpleEdit = () => {
+  const setIsEditPrizeTierModalOpen = useUpdateAtom(isEditPrizeTiersModalOpenAtom)
+  const [prizeTierEdits, setPrizeTierEdits] = useAtom(prizeTierEditsAtom)
+  const [prizeTierHistoryAddress] = useAtom(selectedPrizeTierHistoryAddressAtom)
+  const prizePools = usePrizePools()
+  const [selectedPrizePoolId] = useAtom(selectedPrizePoolIdAtom)
+  const prizePool = prizePools.find((prizePool) => prizePool.id() === selectedPrizePoolId)
+  const { data: tokens, isFetched: isTokensFetched } = usePrizePoolTokens(prizePool)
+  const { data, isFetched: isPrizeTierFetched } = usePrizeTierHistoryData(prizePool)
+  const [isAdvancedDisplay, setAdvancedDisplay] = useState(false)
+
+  const onSubmit = useCallback(
+    (formValues: EditPrizeTierFormValues) => {
+      const newPrizeTierEdits = formatPrizeTierFromFormValues(formValues, tokens?.token.decimals)
+      setPrizeTierEdits((prizeTierEdits) => {
+        // TODO: should check if edits actually make it different from existing data
+        const updatedPrizeTierEdits = { ...prizeTierEdits }
+        if (!updatedPrizeTierEdits[prizePool.chainId]) {
+          updatedPrizeTierEdits[prizePool.chainId] = {}
+        }
+        updatedPrizeTierEdits[prizePool.chainId][prizeTierHistoryAddress] = {
+          ...updatedPrizeTierEdits[prizePool.chainId][prizeTierHistoryAddress],
+          ...newPrizeTierEdits
+        }
+        return updatedPrizeTierEdits
+      })
+      setIsEditPrizeTierModalOpen(false)
+    },
+    [
+      prizePool.chainId,
+      prizeTierHistoryAddress,
+      setIsEditPrizeTierModalOpen,
+      setPrizeTierEdits,
+      tokens?.token.decimals
+    ]
+  )
+
+  const defaultValues = useMemo(() => {
+    const existingEdits =
+      prizeTierEdits?.[prizePool.chainId]?.[prizePool.prizeTierHistoryMetadata.address]
+    if (!!existingEdits) {
+      return formatFormValuesFromPrizeTier(existingEdits, tokens.token.decimals, { round: true })
+    }
+    return formatFormValuesFromPrizeTier(data.upcomingPrizeTier, tokens.token.decimals, {
+      round: true
+    })
+  }, [
+    data.upcomingPrizeTier,
+    prizePool.chainId,
+    prizePool.prizeTierHistoryMetadata.address,
+    prizeTierEdits,
+    tokens.token.decimals
+  ])
+
+  return (
+    <div>
+      <p>Make changes to {getNetworkNiceNameByChainId(prizePool.chainId)}'s Prize Tiers:</p>
```

Comment:
Use the same label as the cards on the main page to keep it consistent for users. Share the `PrizePoolTitle` code.

Reply:
Abstracted out the `PrizePoolTitle` code and used it (slightly enlarged) for the title of the edit modal. (b90f3fe)